startup-script.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```

Comment:
Please follow the shell style guide: https://google.github.io/styleguide/shellguide.html